go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -301,11 +294,6 @@ func (be *BuiltinBackupEngine) executeIncrementalBackup(ctx context.Context, par
 // and an overall error.
 func (be *BuiltinBackupEngine) executeFullBackup(ctx context.Context, params BackupParams, bh backupstorage.BackupHandle) (bool, error) {
 
-	params.Logger.Infof("Hook: %v, Compress: %v", backupStorageHook, backupStorageCompress)
```

Comment:
Same comment as above for the log line.

Reply:
This is a duplicate log line, because the exact same thing was already being logged just before we call this function. So I have left this deleted.